doc/man3/SSL_set1_echstore.pod

```diff
@@ -67,42 +67,120 @@ SSL_CTX_ech_set_callback,SSL_set1_ech_config_list
 
 =head1 DESCRIPTION
 
-TODO(ECH): Complete this text...
+TODO(ECH): Replace XXXX with issued RFC number.
 
 The Encrypted Client Hello (ECH) APIs described here are built around
-the concept of an `OSSL_ECHSTORE` which contains ECH configuration
-information relevant for the current 'SSL_CTX' or 'SSL' connection.
-
-Mention SSL_set1_echstore() is a thing
-Mention OSSL_ECHSTORE_new() is a thing
-Mention OSSL_ECHSTORE_free() is a thing
-Mention OSSL_ECHSTORE_new_config() is a thing
-Mention OSSL_ECHSTORE_write_pem() is a thing
-Mention OSSL_ECHSTORE_read_echconfiglist() is a thing
-Mention OSSL_ECHSTORE_get1_info() is a thing
-Mention OSSL_ECHSTORE_downselect() is a thing
-Mention OSSL_ECHSTORE_set1_key_and_read_pem() is a thing
-Mention OSSL_ECHSTORE_read_pem() is a thing
-Mention OSSL_ECHSTORE_num_keys() is a thing
-Mention OSSL_ECHSTORE_num_entries() is a thing
-Mention OSSL_ECHSTORE_flush_keys() is a thing
-Mention SSL_CTX_set1_echstore() is a thing
-Mention SSL_CTX_get1_echstore() is a thing
-Mention SSL_get1_echstore() is a thing
-Mention SSL_ech_set1_server_names() is a thing
-Mention SSL_ech_set1_outer_server_name() is a thing
-Mention SSL_ech_set1_outer_alpn_protos() is a thing
-Mention SSL_ech_get1_status() is a thing
-Mention SSL_ech_set1_grease_suite() is a thing
-Mention SSL_ech_set_grease_type() is a thing
-Mention SSL_ech_set_callback() is a thing
-Mention SSL_ech_get1_retry_config() is a thing
-Mention SSL_CTX_ech1_set_outer_alpn_protos() is a thing
-Mention SSL_CTX_ech_raw_decrypt() is a thing
-Mention SSL_CTX_ech_set_callback() is a thing
-Mention SSL_set1_ech_config_list() is a thing
-
-=head2 Callback Function
+the concept of an B<OSSL_ECHSTORE> which contains ECH configuration
+information relevant for the current B<SSL_CTX> or B<SSL> connection.
```

Comment:
Not sure what is meant by `the current` here. I assume it is meant for the respective SSL_CTX or SSL connection that has this echstore set but that is IMO obvious.
```suggestion
information relevant for an B<SSL_CTX> or B<SSL> connection.
```

Reply:
did s/the current/an/